homeassistant/components/fortigate/device_tracker.py

```diff
@@ -0,0 +1,110 @@
+"""Device tracker for Fortigate Firewalls."""
+from collections import namedtuple
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_USERNAME, \
+    CONF_PASSWORD, CONF_DEVICES
+from homeassistant.components.device_tracker import DeviceScanner
+from homeassistant.helpers import config_validation as cv
+from homeassistant.components.device_tracker import PLATFORM_SCHEMA
+
+
+from . import DATA_FGT
+
+_LOGGER = logging.getLogger(__name__)
+
+DETECTED_DEVICES = "/monitor/user/detected-device"
+
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_USERNAME): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Optional(CONF_DEVICES, default=[]):
+        vol.All(cv.ensure_list, [cv.string]),
+})
```

Comment:
This looks like duplicate code.

Reply:
one is the CONFIG_SCHEMA the other is PLATFORM_SCHEMA, I referred from the doc

Reply:
We'll only use the `CONFIG_SCHEMA` in the component. We should remove the platform schema here. All config should go under the component.

Reply:
fixed